camp/templates/worldmap/maps/map_worldmap.html

```diff
@@ -0,0 +1,452 @@
+{% include "worldmap/ext_header.html" %}
```

Comment:
I am trying to figure out why you need this. It looks exactly the same as in client/templates/worlmap/maps/map_worldmap.html
Could you explain me why this is needed?

Reply:
Okl, it's unnessary. Because I want to debug the js in geoexplorerr-client.js, I need to use [](https://github.com/GeoNode/geonode/blob/master/geonode/client/templates/worldmap/maps/map_worldmap.html#L3). I have removed the three templates from CAMP.